docs/user_manual/source/exceptions_interrupts.rst

```diff
@@ -98,6 +97,27 @@ In Debug Mode, all interrupts are ignored independent of ``mstatus``.MIE and the
    Load bus fault and store bus fault are handled as imprecise non-maskable interrupts
    (as opposed to precise exceptions).
 
+Non Maskable Interrupts
+-----------------------
+
+NMI's update ``mepc``, ``mcause`` and ``mstatus`` similar to regular interrupts. However, the contents of ``mepc`` is not guaranteed to point to the instruction after the faulted Load or Store.
```

Comment:
NMI's -> NMIs

Reply:
However, the contents of ``mepc`` is not guaranteed to point to the instruction after the faulted Load or Store.
->
However, as the fault that results in NMIs are imprecise, the contents of ``mepc`` is not guaranteed to point to the instruction after the faulted load or store.